bin/improver-spot-extract

```diff
@@ -84,6 +90,24 @@ def main():
         " in altitude to the spot site within the search radius defined when"
         " the neighbour cube was created. May be used with land_constraint.")
 
+    percentile_group = parser.add_argument_group(
+        title="Extract percentiles",
+        description="Extract particular percentiles from probabilistic "
```

Comment:
```suggestion
        description="Extract particular percentiles from probabilistic, percentiles or realizations "
```

Reply:
Done (not quite as above, but percentiles included).